suite-native/device/src/hooks/useDetectDeviceError.tsx

```diff
@@ -152,6 +183,36 @@ export const useDetectDeviceError = () => {
         handleDisconnect,
     ]);
 
+    useEffect(() => {
+        if (hasDeviceError) {
```

Comment:
I'm not sure if `hasDeviceError` is a right condition for such a general error, I don't know if there is always some error 🤔  I feel like `selectIsConnectedDeviceUninitialized` is not robust enough and should not return true if there is such an error (if device.features are undefined)

Reply:
We should be aware of other cases, see for example https://github.com/trezor/trezor-suite/blob/develop/suite-common/suite-utils/src/device.ts#L31 So far "no-seed" alert is the default that catches all those cases, I believe that this new more generic error would be more appropriate.

Reply:
after 4d008a03ad92d85eae17e1439135106a1a77facf it should work, but it should be refactored soon, selectIsConnectedDeviceUninitialized is confusing and it works only because unacquired device is handled with higher priority